src/lib/containers/download_list/DownloadListTable.tsx

```diff
@@ -204,6 +221,107 @@ export default function DownloadListTable(props: DownloadListTableProps) {
     }
   }
 
+  const sortColumn = async (column: string) => {
+    try {
+      setIsLoading(true)
+
+      const isDescending =
+        column === sortedColumn.column ? !sortedColumn.isDescending : true
```

Comment:
lets change this so that by default the column is ascending -

```jsx
  const isDescending =
        column === sortedColumn.column ? !sortedColumn.isDescending : false
```